apps/expo/src/components/OutlinedButtons.tsx

```diff
@@ -0,0 +1,27 @@
+import React from "react";
+import { Text, TouchableOpacity } from "react-native";
+
+interface OutlinedButtonProps {
+  onPress: () => void;
+  children: React.ReactNode;
+  fill: boolean;
+  // icon to be added
```

Comment:
why does the icon need to be added? it should be able to be inserted as a child like `<Button>Baker <Icon /> </Button`